README.md

```diff
@@ -262,6 +274,29 @@ FIRETWEET_MAIN_DIR=/path/to/firetweet/src/main make android-lib
 You can also override this environment variable if you want to use the
 [Flashlight Android Tester](https://github.com/getlantern/lantern-mobile-single-app-example) app.
 
+#### Creating the Android library without docker
+
+1. Install Java JDK 7 or 8
+2. Install [Android SDK Tools](http://developer.android.com/sdk/index.html#Other)
+3. Install NDK(http://developer.android.com/ndk/downloads/index.html)
+4. Install lantern's [fork of gomobile](https://github.com/getlantern/mobile)
```

Comment:
The purpose of lantern's fork of `gomobile` is to skip iOS related tools right? It would be good to have instructions to install it. I encountered following error.

```bash
 ~/workspace/getlantern/lantern (valencia_ox) $ go get github.com/getlantern/mobile/cmd/gomobile
package github.com/getlantern/mobile/cmd/gomobile: code in directory /Users/merlin/workspace/go/src/github.com/getlantern/mobile/cmd/gomobile expects import "golang.org/x/mobile/cmd/gomobile"
```

I removed `golang.org/x/mobile` directory and built aar sucessfully using following command sequence (with XCode installed).

```bash
go get -v golang.org/x/mobile/cmd/gomobile
. setenv.sh
gomobile init -v
make android-lib-local
```